server/rematch/celery.py

```diff
@@ -6,6 +6,8 @@
 
 # set the default Django settings module for the 'celery' program.
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'rematch.settings')
+import django
+django.setup()
```

Comment:
Can you explain why this is required?
I'm running `celery -A rematch.celery worker --loglevel=info` from `./server/` and everything seems to work without it.